nntrainer/tensor/swap_device.h

```diff
@@ -18,15 +18,22 @@
 #include <map>
 #include <memory>
 #include <string>
-#include <sys/mman.h>
 #include <sys/stat.h>
 #include <sys/types.h>
 #include <system_error>
-#include <unistd.h>
 #include <utility>
 
-/* Uncomment this to use mmap for swap data */
-#define USE_MMAP
+#if defined(_WIN32)
```

Comment:
The default value of swap device should be  USE_MMAP. I think if this feature is not supported in Windows, it might be better not to compile this swap_device.h